### PR TITLE
Create a database based on a diagram: Clinic database

### PR DESCRIPTION
**Pull request by KIsmoilov**
The following tasks have been completed:

- Created a file named `schema_based_on_diagram.sql` where we implemented the database from the diagram.
- We created join tables from many-to-many relationships.
- Added the FK indexes.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  date_of_birth DATE,
+  PRIMARY KEY(id)
+  );
+
+CREATE TABLE medical_histories (
+     id INT GENERATED ALWAYS AS IDENTITY,
+     admitted_at timestamp,
+     patient_id INT,
+     status VARCHAR(100),
+     FOREIGN KEY (patient_id) REFERENCES patients(id),
```
Comment (Felix45) [inline schema_based_on_diagram.sql:13]: - [ Required ] Great job adding the foreign key to this table, however, you forgot to add an `index` to the foreign key column. Please consider adding an index on all the foreign key columns as specified in the project requirements as shown below.

Screenshot from 2023-01-24 14-51-02

You can achieve this by using the SQL query syntax below.
```SQL
  CREATE INDEX ON medical_histories (patient_id);
```

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+     PRIMARY KEY(id)
+);
+
+CREATE TABLE invoices(
+   id INT GENERATED ALWAYS AS IDENTITY,
+    total_amount DECIMAL,
+    generated_at TIMESTAMP, 
+    payed_at TIMESTAMP, 
+    medical_history_id INT,
+    FOREIGN KEY(medical_history_id) REFERENCES medical_histories(id);
```
Comment (Felix45) [inline schema_based_on_diagram.sql:23]: - [ Required ] Great job adding the foreign key to this table, however, you forgot to add an `index` to the foreign key column. Please consider adding an index on all the foreign key columns as specified in the project requirements as shown below.

Screenshot from 2023-01-24 14-51-02

You can achieve this by using the SQL query syntax below.
```SQL
  CREATE INDEX ON invoices (medical_history_id);
```

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @KIsmoilov  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :heavy_check_mark: 
- [ ] Good SQL query design :white_check_mark: 
- [ ] GitHub flow is correctly used :white_check_mark: 
- [ ] Descriptive pull request title and summary :white_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @KIsmoilov  @Felix45 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**